### ISSUE TITLE
Voice options?

### ISSUE THREAD

**Issue by jmoldyvan** [XollabOS/talktometechnology#36]
Would there be a consideration for different voice options?

It wouldn't be too difficult, with the current code, to add a list of voice options (even some with accents!) The speech synthesis method has a built in list, speechSynthesis.getVoices(), with some minor alterations the user could even choose the preferred voice.

I'd be happy to try and implement it/make a pull request.

**Comment by dnsmichi** [comment]
@jmoldyvan Sounds like a good idea, thanks :-) 

I'm wondering how to make it configurable with the current code base, and I remember that @codebymatthew @laurelayounis wanted to change the code base, maybe with also providing this possibility. Y'all - thoughts?

**Comment by codebymatthew** [comment]
That is the eventual goal.  Voice interaction and eye tracking to select tiles is also something we are looking into.

**Comment by laurelayounis** [comment]
yes this is the plan, we saw the pull request adding the different voices which is amazing! It has some merge conflicts that we need to look into before merging it, we're also trying to determine which direction we're going in for re-designing the code base. We'll go ahead and keep this issue open until we determine the best route to solve it. Thank you @jmoldyvan & @dnsmichi for all of your help!

**Comment by jmoldyvan** [comment]
Thank you for checking it out and considering the changes! If there's any alterations needed that I can make to the pull request/merge conflicts to increase the possibility of merging, let me know.  Hope the redesign goes well.
